tests/performance/local/future_overhead.cpp

```diff
@@ -52,10 +83,17 @@ std::uint64_t num_iterations = 0;
 ///////////////////////////////////////////////////////////////////////////////
 double null_function()
 {
-    double d = 0.;
-    for (std::uint64_t i = 0; i < num_iterations; ++i)
-        d += 1. / (2. * i + 1.);
-    return d;
+    if (num_iterations>0) {
+        const int array_size = 4096;
+        std::array<double, array_size> dummy;
+        for (std::uint64_t i = 0; i < num_iterations; ++i) {
+            for (std::uint64_t j = 0; j < array_size; ++j) {
+                dummy[j] = 1.0 / (2.0 * i * j + 1.0);
+            }
+        }
+        return dummy[0];
+    }
+    return 0.0;
```

Comment:
Did this change just to have some more work?

Reply:
yes. When the num_iterations>0 the work done in the loop is much more substantial. The reason for this is to allow testing of memory consumption. If many tasks have stacks attached on creation (they should not) then when thousands are siting in waiting queues, the memory explodes. This is just to allow me to test not just task overheads, but also memory overheads and catch problems when testing new lazy stack init type PRs